src/index.html

```diff
@@ -1,27 +1,437 @@
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    />
     <link
       rel="preconnect"
-      href="https://fonts.gstatic.com"
+      href="https://fonts.googleapis.com"
     />
     <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
-      rel="stylesheet"
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
     />
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="styles/index.scss"
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        data-qa="card"
+        href="#Home"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#Apple"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Samsung"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Smartphones"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Laptops&Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Gadgets"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#TABLETS"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Photo"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#Video"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <section
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image">
+          <img
+            src="./images/imac.jpeg"
+            alt="iMac photo"
+            class="card__photo"
+          />
+        </div>
+        <h1 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
```

Comment:
This tag used only for title on the page, so use the another tag here, fix it everywhere
```suggestion
        <h2 class="card__name">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
```

Reply:
I feel like if I remove this div, the purpose of this image will be unclear.